src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md

```diff
@@ -36,12 +36,34 @@ New-AzureRmVmss [[-ResourceGroupName] <String>] [-VMScaleSetName] <String> [-AsJ
 
 ## DESCRIPTION
 The **New-AzureRmVmss** cmdlet creates a Virtual Machine Scale Set (VMSS) in Azure.
-This cmdlet takes a **VirtualMachineScaleSet** object as input.
+This cmdlet takes a **VirtualMachineScaleSet** object as input or the **`SimpleParameterSet`** can be used to create the vmss with a pre-set configuration which along with the vmss creates other resources required by the vmss as well.
 
 ## EXAMPLES
 
-### Example 1: Create a VMSS
+### Example 1: Create a VMSS using the **`SimpleParameterSet`**
+```powershell
+$vmssName = <VMSSNAME>
+# Create credentials, I am using one way to create credentials, there are others as well. 
+# Pick one that makes the most sense according to your use case.
+$vmPassword = ConvertTo-SecureString <PASSWORD_HERE> -AsPlainText -Force
+$vmCred = New-Object System.Management.Automation.PSCredential(<USERNAME_HERE>, $vmPassword)
+
+#Create a VMSS using the default settings
+new-azurermvmss -Credential $vmCred -VMScaleSetName $vmssName 
 ```
+
+The command above creates the following with the name`$vmssName` :
+* A ResourceGroup
+* A virtual network
+* A load balancer
+* A public IP
+* the VMSS with 2 instances
+
+The default image chosen for the vms in the vmss is `2016-Datacenter Windows Server` and the SKU is `Standard_DS1_v2`
```

Comment:
@praries880 nit: let's capitalize any acronyms: "vms" --> "VMs" and "vmss" --> "VMSS"